src/puyapy-stubs/_unsigned_builtins.pyi

```diff
@@ -10,12 +10,14 @@ from typing import TypeVar, overload
 
 from puyapy import UInt64
 
-@overload
-def urange(__stop: int | UInt64) -> Iterator[UInt64]: ...
-@overload
-def urange(
-    __start: int | UInt64, __stop: int | UInt64, __step: int | UInt64 = ...
-) -> Iterator[UInt64]: ...
+class urange(Iterable[UInt64]):
+    @overload
+    def __init__(self, stop: int | UInt64, /) -> None: ...
+    @overload
+    def __init__(self, start: int | UInt64, stop: int | UInt64, /) -> None: ...
+    @overload
+    def __init__(self, start: int | UInt64, stop: int | UInt64, step: int | UInt64, /) -> None: ...
+    def __iter__(self) -> Iterator[UInt64]: ...
```

Comment:
`__iter__` not required, it's inherited from `Iterable[UInt64]`

Reply:
That was my impression too however I get this:
```
scratch_slots\contract.py:10: error: Cannot instantiate abstract class "urange" with abstract attribute "__iter__"  [abstract]
            for i in urange(3, 20):
```

Reply:
oh weird, it's declared as a Protocol in typeshed.
oh well, doesn't hurt to have it